### DISCUSSION TITLE
Chrome opens without existing Chrome profiles on PC

### DISCUSSION THREAD

**Discussion by jtresko** [f1multiviewer/issue-tracker#153] [Q&A] [answered]
When Chrome opens to log in, it is loaded with a 'Person 1' profile, and none of my existing chrome profiles (with saved f1tv password) are available in the chrome window.

passwords do not save to this Person 1 profile.

![image](https://user-images.githubusercontent.com/86134711/183920062-726a51b8-9c55-490e-a4d2-ab3ae94663a7.png)

Is there a Setting somewhere in Chrome or MV?

**Comment by f1multiviewer** [comment, marked as answer]
That's a F1MV setting. I initially chose to do it this way, so F1MV doesn't accidentally access anything personal, and only accesses F1TV-related stuff. However, I will likely make sure the user profile directory is kept so saved passwords persist for the next time you sign in.

**Reply by jtresko** [reply]
Is this a setting in your backend or is there a way to edit this in config files or UI?

Perhaps just being able to open the login screen in a new tab within in an existing Chrome window would be simplest.

I recently created a batch script to launch companion apps when I launch my racing simulator. One of them is a web app and was able to configure some commands in launch options to use chrome defaults... Which enabled it to open the web app with active sessions.

`chrome_proxy.exe --profile-directory=Default --app-id=iajpfcheplcajlodcplkophbmmojdgkk`
